pkgs/mo2installer/default.nix

```diff
@@ -0,0 +1,58 @@
+{
+  fetchFromGitHub,
+  lib,
+  pkgs,
```

Comment:
Please include the needed packages verbatim. Including `pkgs` takes a toll on evaluation.

Reply:
Huh, didn't know that.